### PR TITLE
Draft of descriptive statistics

### PR DESCRIPTION
**Pull request by realtommypoole**
Added a few initial descriptive statistics analyses to the dataset in a new file for review.

### CODE REVIEW

File: projects/project1/Descriptive Statistics Draft.ipynb [hunk 1]
```diff
@@ -0,0 +1,440 @@
...
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Import standard libraries\n",
+    "import numpy as np\n",
+    "import pandas as pd\n",
+    "from sklearn.datasets import fetch_california_housing\n",
```
Comment (Fatimat01) [inline projects/project1/Descriptive Statistics Draft.ipynb:26]: our data isnt for California housing

File: projects/project1/Descriptive Statistics Draft.ipynb [hunk 2]
```diff
@@ -0,0 +1,440 @@
...
+      "1 2010-12-01 08:26:00       3.39     17850.0  United Kingdom  \n",
+      "2 2010-12-01 08:26:00       2.75     17850.0  United Kingdom  \n",
+      "3 2010-12-01 08:26:00       3.39     17850.0  United Kingdom  \n",
+      "4 2010-12-01 08:26:00       3.39     17850.0  United Kingdom  \n"
+     ]
+    }
+   ],
+   "source": [
+    "# Read the Excel file\n",
+    "df = pd.read_excel('datasets/online+retail/Online Retail.xlsx', engine='openpyxl')\n",
```
Comment (Fatimat01) [inline projects/project1/Descriptive Statistics Draft.ipynb:58]: is this assuming you have the dataset locally?

File: projects/project1/Descriptive Statistics Draft.ipynb (hunk 1 above)
Comment (realtommypoole) [reply projects/project1/Descriptive Statistics Draft.ipynb:26]: Good catch, I'll change this. You are correct, this was lingering from the lab 3 code.

File: projects/project1/Descriptive Statistics Draft.ipynb (hunk 2 above)
Comment (realtommypoole) [reply projects/project1/Descriptive Statistics Draft.ipynb:58]: This is assuming the dataset is in the repository files of the project repository. Happy to do it another way if you'd prefer.

**Comment by realtommypoole** [comment]
I will create a new pull request with the fetch california housing removed.

**Pull request closed without merging**